ketchup/main.py

```diff
@@ -60,7 +60,7 @@ def determine_end_and_start_date():
 # TODO: Re-add the "Assign both a limit to how many episodes a day you watch,
 #       as well as how many days a week you decide to watch.
 def run():
-    print("\nHello and welcome to AnimeKetchup "
+    print("\nHello and welcome to animeketchup "
```

Comment:
i mean sure but why lol

Reply:
PyCharm refactoring for you. Changes any instance of "AnimeKetchup" to "animeketchup" instead. It wasn't me being picky this time around haha.